misc/test-lib/helpers.js

```diff
@@ -1,25 +1,57 @@
 // jasmine matcher for expecting an element to have a css class
 // https://github.com/angular/angular.js/blob/master/test/matchers.js
 beforeEach(function() {
-  this.addMatchers({
-    toHaveClass: function(cls) {
-      this.message = function() {
-        return "Expected '" + this.actual + "'" + (this.isNot ? ' not ' : ' ') + "to have class '" + cls + "'.";
-      };
+  jasmine.addMatchers({
+    toHaveClass: function(util, customEqualityTesters) {
+      return {
+        compare: function(actual, expected) {
+          var result = {
+            pass: actual.hasClass(expected)
+          };
 
-      return this.actual.hasClass(cls);
+          if (result.pass) {
+            result.message = 'Expected ' + actual + ' not to have the ' + expected + ' class.';
+          } else {
+            result.message = 'Expected ' + actual + ' to have the ' + expected + ' class.';
```

Comment:
With parentheses and whitespace, I think it's reasonable. See the original message above.
Also note that there were quotes (`'`) around the `actual` and `expected` before this change.